pkg/client/tsconfig.json

```diff
@@ -29,6 +29,7 @@
     },
     "skipLibCheck": true,
     "strict": true, // Includes alwaysStrict and noImplicitAny
+    "noImplicitAny": false,
```

Comment:
We need to avoid going on that road.

Reply:
This is the approach that forklift-ui takes. Need this to manipulate the global window object.